test/cxx/stl/vector_pair/Makefile

```diff
@@ -0,0 +1,27 @@
+#
+# Makefile
+#
+# makefile: vector
+#
+# Copyright (C) 2017-2024 Tactical Computing Laboratories, LLC
+# All Rights Reserved
+# contact@tactcomplabs.com
+#
+# See LICENSE in the top level directory for licensing details
+#
+
+.PHONY: src
+
+EXAMPLE=vector_pair
+CC=riscv64-unknown-elf-g++
+#CC="${RVCC}"
+#ARCH=rv64g
+ARCH=rv64imafdc
```

Comment:
gc

Reply:
These are all from Forza -- I didn't change them except for some `.c` filenames, a librev library path passed to SST, and the `revalloc.h` common header.